backend/lib/server.js

```diff
@@ -21,19 +21,22 @@ function toMilliseconds (seconds) {
 function createServer (app) {
   const port = app.get('port')
   const periodSeconds = app.get('periodSeconds')
-  const healthCheck = app.get('healthCheck')
+  const healthCheckFunc = app.get('healthCheck')
   const logger = app.get('logger')
   const hooks = app.get('hooks')
 
   // create server
   const server = http.createServer(app)
 
   // create terminus
+  const healthChecks = {}
+  if (typeof healthCheckFunc === 'function') {
```

Comment:
why do we need this check?

Reply:
The API/backend and metrics apps are both passed to this function for creating the server (this approach is open for debate). For the metrics endpoint we do not have/want a health check. So to make this `createServer`-logic reusable we only add the health check routes if the `app` has defined a health check function.